test/system/dashboard/projects_test.rb

```diff
@@ -101,6 +101,7 @@ def setup
       end
       fill_in I18n.t(:'dashboard.projects.index.search.placeholder'), with: projects(:project1).name
 
+      sleep 1
```

Comment:
Instead of adding sleep can you change this test to as follows:

```ruby
test 'can filter and then sort the list of projects' do
      visit dashboard_projects_url

      assert_text 'Displaying items 1-20 of 38 in total'
      assert_selector 'h1', text: I18n.t(:'dashboard.projects.index.title')
      assert_selector 'tr', count: 20
      within first('tr') do
        assert_text projects(:project1).human_name
      end
      fill_in I18n.t(:'dashboard.projects.index.search.placeholder'), with: projects(:project1).name

      assert_text 'Displaying 12 items'
      assert_selector 'tr', count: 12
      assert_no_selector 'a', text: /\A#{I18n.t(:'components.pagination.next')}\Z/
      assert_no_selector 'a', text: I18n.t(:'components.pagination.previous')

      click_on I18n.t(:'dashboard.projects.index.sorting.updated_at_desc')
      click_on I18n.t(:'dashboard.projects.index.sorting.namespace_name_desc')
      assert_no_text I18n.t(:'dashboard.projects.index.sorting.updated_at_desc')
      assert_text I18n.t(:'dashboard.projects.index.sorting.namespace_name_desc')

      assert_selector 'tr', count: 12
      within first('tr') do
        assert_text projects(:project19).human_name
      end
    end
```

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/581/commits/38d841c23b75f08484fd2c12e4ad50afae524697